Traceback (most recent call last):
  File "<ipython-input-2-a68727528259>", line 2, in <module>
    raise Exception('This is the error message.')
Exception: This is the error message.
